src/app/core/entity/entity-mapper.service.ts

```diff
@@ -94,7 +94,7 @@ export class EntityMapperService {
       entity
     );
     const result = await this._db.put(rawData, forceUpdate);
-    if (result.ok) {
+    if (result?.ok) {
```

Comment:
result can be `null`?

Reply:
Apparently. I think this causes #515. It's the only time we access a variable called 'ok'.